imagetest/test_suites/image_boot/image_boot_test.go

```diff
@@ -17,11 +17,94 @@ import (
 	"github.com/GoogleCloudPlatform/guest-test-infra/imagetest/utils"
 )
 
+// The values have been decided based on running spot tests for different images.
+var imageFamilyBootTimeThresholdMap = map[string]float64{
+	"centos":      60,
+	"debian":      50,
+	"rhel":        60,
+	"rocky-linux": 60,
+	"sles-12":     85,
+	"sles-15":     120,
+	"ubuntu":      75,
+	"ubuntu-pro":  110,
+}
+
 const (
 	markerFile     = "/boot-marker"
 	secureBootFile = "/sys/firmware/efi/efivars/SecureBoot-8be4df61-93ca-11d2-aa0d-00e098032b8c"
 )
 
+// Returns the threshold value
+func getThresholdValue(image string) float64 {
+	switch {
+	case strings.Contains(image, "centos"):
+		return imageFamilyBootTimeThresholdMap["centos"]
+	case strings.Contains(image, "debian"):
+		return imageFamilyBootTimeThresholdMap["debian"]
+	case strings.Contains(image, "rhel"):
+		return imageFamilyBootTimeThresholdMap["rhel"]
+	case strings.Contains(image, "rocky-linux"):
+		return imageFamilyBootTimeThresholdMap["rocky-linux"]
+	case strings.Contains(image, "sles-12"):
+		return imageFamilyBootTimeThresholdMap["sles-12"]
+	case strings.Contains(image, "sles-15"):
+		return imageFamilyBootTimeThresholdMap["sles-15"]
+	case strings.Contains(image, "ubuntu-pro"):
+		return imageFamilyBootTimeThresholdMap["ubuntu-pro"]
+	case strings.Contains(image, "ubuntu"):
+		return imageFamilyBootTimeThresholdMap["ubuntu"]
+	default:
+		return 0
```

Comment:
i think the default should be an explicit error that this is an unsupported OS, or a reasonable default. it might mean moving the case statement into the main function